ports/glslang/vcpkg.json

```
@@ -25,7 +25,7 @@
       "description": "Build with dynamic typeinfo"
     },
     "tools": {
-      "description": "Build the glslangValidator and spirv-remap binaries",
+      "description": "Build the glslang binary",
       "supports": "!ios"
```

```suggestion
      "description": "Build the glslang binaries",
      "supports": "!android & !ios"
```
AFAICS it is still multiple binaries.
And now zero for android. (?)

---

> Suggested change
>       "description": "Build the glslang binary",
>       "supports": "!ios"
>       "description": "Build the glslang binaries",
>       "supports": "!android & !ios"
> 
> AFAICS it is still multiple binaries. 

`ENABLE_GLSLANG_BINARIES` simply ensures that it executes the StandAlone-CMakeLists. This builds a `glslang-standalone` executable, which is given the output name `glslang`. In addition, a symlink `glslangValidator` is created for `glslang`. Under Windows or MinGW, you are right: there are two targets, because `glslangValidator` is a copy instead of a symlink. Under the other platforms, there is only one, because `spirv-remap` has been moved to another project. Since `glslangValidator` is only intended to be a symlink or copy for legacy reasons, I found the singular form more appropriate here.

see also the [option description](https://github.com/KhronosGroup/glslang/blob/b5782e52ee2f7b3e40bb9c80d15b47016e008bc9/CMakeLists.txt#L97) of them.

> And now zero for android. (?)

I'm still wondering why it still worked with Android in 15.1.0 as the are turning off `ENABLE_GLSLANG_BINARIES` [since 14.0.0](https://github.com/KhronosGroup/glslang/commit/a7b18c08d0a9d1b98444adb2dcaa857ee5413dc9#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a)

---

> In addition, a symlink `glslangValidator` is created for `glslang`. Under Windows or MinGW, you are right: there are two targets, because `glslangValidator` is a copy instead of a symlink. Under the other platforms, there is only one, because `spirv-remap` has been moved to another project. Since `glslangValidator` is only intended to be a symlink or copy for legacy reasons, I found the singular form more appropriate here.

These type of executables do different things, depending on the name. So removing one name is taking away ready-to-use capabilities.